samples/CoreAnimationTest/CoreAnimationTest/ProgrammaticViewController.m

```diff
@@ -38,13 +38,13 @@ - (void)viewDidLoad {
     [self addMenuItemViewController:[[CALayerAnimationViewController alloc] init] andTitle:@"CALayer Animation (CAAnimation)"];
 
     // CAShapeLayer
-    [self addMenuItemSegue:@"CAShapeLayer" andTitle:@"CAShapeLayer"];
+    [self addMenuItemSegue:@"CAShapeLayer" andTitle:@"CAShapeLayer - not supported"];
```

Comment:
Filed #1942 to capture differences - removing the "not supported" text on the CAShapeLayer menu entry.

---
In reply to: [99848734](https://github.com/Microsoft/WinObjC/pull/1874#discussion_r99848734) [](ancestors = 99848734,99437888)